internal/dependencymanager/add.go

```diff
@@ -58,7 +59,7 @@ func add(
 
 	dep := args[0]
 
-	installer, err := NewDependencyInstaller(logger, state, addFlags.skipDeployments)
+	installer, err := NewDependencyInstaller(logger, state, addFlags.skipDeployments, addFlags.skipAlias)
```

Comment:
minor, but should we just have `NewDependencyInstaller` take `addFlagsCollection` as a whole instead? That way we don't have to interrupt the method parameters each time if we want to add/remove params related to flags

Reply:
Can do that currently @ianthpun . Only time that could become a problem is if the install flags and add flags start to deviate in anyway. Thoughts?

Reply:
i guess the option in that case would be to have a "dependencyManageConfig" struct, which add/install would fill in. Might be overkill, but something to consider. I'd prefer that over using the existing add/installFlag structs